templates/landscape/install.html

```diff
@@ -752,29 +567,28 @@ <h3 class="p-heading--2">Configure</h3>
             <p class="p-step-counter"></p>
           </div>
           <div class="col-9 col-medium-5">
-            <p class="p-heading--5">Define variables using Jinja</p>
+            <p class="p-heading--5">To install landscape-client from a PPA</p>
             <div class="p-code-snippet">
-              <pre class="p-code-snippet__block is-wrapped"><code># LANDSCAPE_ACCOUNT_NAME: self-hosted Landscape users can leave this as `standalone`
-# Landscape SaaS users can find the account name here: https://landscape.canonical.com/settings
-{% raw %}{% set LANDSCAPE_ACCOUNT_NAME = 'standalone' %}{% endraw %}
-
-# LANDSCAPE_FQDN: `landscape.canonical.com` is only for Landscape SaaS users
-# Fully Qualified Domain Name of your Landscape instance
-{% raw %}{% set LANDSCAPE_FQDN = 'landscape.canonical.com' %}{% endraw %}
-
-# LANDSCAPE_COMPUTER_TITLE: default value is fine
-# human readable name in Landscape
-{% raw %}{% set LANDSCAPE_COMPUTER_TITLE = "dedicated server" %}{% endraw %}</code></pre>
+              <pre class="p-code-snippet__block is-wrapped"><code>apt:
+  sources:
+    trunk-testing-ppa:
+      source: ppa:landscape/self-hosted-23.03
+            </code></pre>
             </div>
           </div>
           <hr class="u-no-margin--bottom u-hide--small">
           <div class="col-3 col-medium-1 u-hide--small">
             <p class="p-step-counter"></p>
           </div>
           <div class="col-9 col-medium-5">
-            <p class="p-heading--5">Install within the runcmd section of Cloud Init</p>
+            <p class="p-heading--5">Configure landscape-client</p>
+            <p>To discover additional supported client keys visit the <a href="https://cloudinit.readthedocs.io/en/latest/reference/modules.html#landscape">cloud-init Landscape module documentation</a>.</p>
```

Comment:
I had omitted this in the copydoc originally, and I added it in when I noticed it was missing:
`Self-hosted Landscape users should set account_name as `standalone` and define the url and ping_url keys. Landscape SaaS customers should specify their <a href="https://landscape.canonical.com/settings">account name</a>, and the url and ping_url keys can be omitted.`